modules/accredited_representative_portal/app/sidekiq/accredited_representative_portal/github_csv_fetcher.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module AccreditedRepresentativePortal
+  class GithubCsvFetcher
+    def fetch
+      Rails.logger.info "Fetching CSV from GitHub: #{repo} path: #{path}"
+      content = client.contents(repo, path: path).content
+      decoded_content = Base64.decode64(content)
+      raise CSV::MalformedCSVError.new('Empty CSV file', 0) if decoded_content.blank?
```

Comment:
@ojbucao Could we unify the 0 and max row count exceptional cases rather than claim malformed?